AndroidNativeLoginTemplate/app/src/main/java/com/salesforce/androidnativelogintemplate/NativeLogin.kt

```diff
@@ -512,15 +492,21 @@ class NativeLogin : ComponentActivity() {
                         elevation = CardDefaults.cardElevation(defaultElevation = 6.dp)
                     ) {
                         Spacer(modifier = Modifier.height(25.dp))
-                        Image(
-                            painter = painterResource(id = sf__salesforce_logo),
-                            colorFilter = ColorFilter.tint(colorScheme.primary),
-                            contentDescription = "",
-                            modifier = Modifier.align(CenterHorizontally),
-                        )
+                        if (identityFlowLayoutTypeActive != StartRegistration) {
```

Comment:
There's an issue with the IME layout of the native login view where tall layouts don't correctly scroll to the focused element.  I've been trying to fix it, but it certainly looks worth its own bug in the interest of cycle time and small work units.  In the meantime, it's not so bad if the rather large SF logo isn't displayed for start registration.

I'll get that new work item in for a better IME layout.

Reply:
The follow-up is `W-15811064`.